code/game/objects/items/storage/boxes.dm

```diff
@@ -768,6 +768,16 @@
 	for(var/i in 1 to 6)
 		new /obj/item/ammo_casing/shotgun/beanbag(src)
 
+/obj/item/storage/box/rubber
+	name = "box of rubber shot"
+	desc = "A box full of rubber shot shells."
+	icon_state = "rubbershot_box"
+	illustration = null
+
+/obj/item/storage/box/rubber/PopulateContents()
+	for(var/i in 1 to 7)
+		new /obj/item/ammo_casing/shotgun/rubbershot(src)
+
```

Comment:
We already have a `/obj/item/storage/box/rubbershot` type with the same purpose. Please remove this duplicate.